test/cli_update_test.go

```diff
@@ -183,3 +184,45 @@ func (suite *PouchUpdateSuite) TestUpdateContainerWithoutFlag(c *check.C) {
 
 	command.PouchRun("rm", "-f", name).Assert(c, icmd.Success)
 }
+
+// TestUpdateContainerEnv is to verify the correctness of updating env of container.
+func (suite *PouchUpdateSuite) TestUpdateContainerEnv(c *check.C) {
+	name := "update-container-env"
+
+	command.PouchRun("create", "-m", "300M", "--name", name, busyboxImage).Assert(c, icmd.Success)
+
+	command.PouchRun("update", "--env", "foo=bar", name).Assert(c, icmd.Success)
+
+	output := command.PouchRun("inspect", name).Stdout()
+	result := []types.ContainerJSON{}
+	if err := json.Unmarshal([]byte(output), &result); err != nil {
+		c.Errorf("failed to decode inspect output: %v", err)
+	}
+
+	if !utils.StringInSlice(result[0].Config.Env, "foo=bar") {
+		c.Errorf("expect 'foo=bar' in container env, but got: %v", result[0].Config.Env)
+	}
+
+	command.PouchRun("rm", "-f", name).Assert(c, icmd.Success)
+}
+
+// TestUpdateContainerLabel is to verify the correctness of updating label of container.
+func (suite *PouchUpdateSuite) TestUpdateContainerLabel(c *check.C) {
+	name := "update-container-label"
+
+	command.PouchRun("run", "-d", "-m", "300M", "--name", name, busyboxImage).Assert(c, icmd.Success)
+
+	command.PouchRun("update", "--label", "foo=bar", name).Assert(c, icmd.Success)
+
+	output := command.PouchRun("inspect", name).Stdout()
+	result := []types.ContainerJSON{}
+	if err := json.Unmarshal([]byte(output), &result); err != nil {
+		c.Errorf("failed to decode inspect output: %v", err)
+	}
+
+	if v, ok := result[0].Config.Labels["foo"]; !ok || v != "bar" {
+		c.Errorf("expect 'foo=bar' in Labels, got: %v", result[0].Config.Labels)
+	}
+
+	command.PouchRun("rm", "-f", name).Assert(c, icmd.Success)
+}
```

Comment:
Could you add an integration test to verify updating env for a running container? @HusterWan

Reply:
ok!